x-pack/platform/plugins/shared/lens/public/persistence/lens_document_service.ts

```diff
@@ -33,46 +31,40 @@ interface ILensDocumentService {
 }
 
 export class LensDocumentService implements ILensDocumentService {
-  private client: VisualizationClient<'lens', LensSavedObjectAttributes>;
+  private client: LensClient;
 
-  constructor(cm: ContentManagementPublicStart) {
-    this.client = getLensClient(cm);
+  constructor(http: HttpStart) {
+    this.client = new LensClient(http);
```

Comment:
Lens client now uses `http` instead of `cm` to call new APIs directly for CRUD operations.